erpnext/stock/doctype/product_recall_notice/product_recall_notice.py

```diff
@@ -23,27 +24,40 @@ def create_stock_entry_from_product_recall_notice(product_recall_notice):
 	"""
 	product_recall_notice = json.loads(product_recall_notice)
 	se_items = []
-	stock_entry_doc = frappe.new_doc("Stock Entry")
-
-	if product_recall_notice.get("recall_from") == "Warehouse":
-		stock_entry_doc.stock_entry_type = "Material Transfer"
-	else:
-		stock_entry_doc.stock_entry_type = "Material Receipt"
 
 	for item in product_recall_notice.get("items"):
-		se_items.append({
-			"item_code":item.get("item_code"),
-			"qty": item.get("qty"),
-			"batch_no": item.get("batch_no"),
-			"s_warehouse": item.get("warehouse") if stock_entry_doc.stock_entry_type == "Material Transfer" else None,
-			"t_warehouse": product_recall_notice.get("recall_warehouse")
+		if item.get("qty") != item.get("received_qty"):
```

Comment:
@MyuddinKhatri where are the comments? Also, why is this being merged into may-sprint-2?